### PR TITLE
Expanded Readme & CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Kilo59**
Expand Readme and add `CONTRIBUTING.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,39 @@
...
+```
+
+#### Set env variables
+
+`GX_CLOUD_ACCESS_TOKEN`
+`GX_CLOUD_ORGANIZATION_ID`
+
+### Start the Agent
+
+May want to run the agent outside docker if testing against a localhost backend or datasource.
```
Comment (NathanFarmer) [inline README.md:29]: I think you need to do this unless you alter the `Dockerfile` somehow.

**Review by NathanFarmer — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Kilo59) [inline README.md:29]:
```suggestion
If you intend to run the agent against local services (Cloud backend or datasources) you will need to run the agent outside of the container.
```

File: README.md [hunk 2]
```diff
@@ -6,6 +6,54 @@
...
+pip install great_expectations_cloud
+```
+##### Optional Dependencies
+```console
+pip install 'great_expectations_cloud[sql]'
+```
+
+```console
+$ gx-agent --help
+usage: gx-agent [-h] [--log-level LOG_LEVEL] [--skip-log-file SKIP_LOG_FILE] [--log-cfg-file LOG_CFG_FILE] [--version]
```
Comment (Kilo59) [inline README.md:24]: We need to use more lazy imports or something, showing this help page is unbearable slow.
On my machine it takes like 5+ seconds

File: README.md [hunk 3]
```diff
@@ -6,8 +6,55 @@
...
+```
+
+#### Set env variables
+
+`GX_CLOUD_ACCESS_TOKEN`
+`GX_CLOUD_ORGANIZATION_ID`
+
+### Start the Agent
+
+If you intend to run the agent against local services (Cloud backend or datasources) run the agent outside of the container.
```
Comment (JennyTee) [inline README.md:44]: Nit: we might consider moving this instruction to the CONTRIBUTING.md file so it doesn't confuse customers trying to start their agent

Comment (Kilo59) [reply README.md:44]: I'm fine with that

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Contributing to `great_expectations_cloud`
+
+## Managing Dependencies
+
+Dependencies are the virtual environment is managed with [`poetry`](https://python-poetry.org/docs/)
```
Comment (Kilo59) [inline CONTRIBUTING.md:5]: Oh wow...🤦

**Pull request merged**